packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -241,6 +241,11 @@ class App extends Component {
           this.setFeature(f.default)
         );
         break;
+      case 'dynamic-import':
```

Comment:
How does this even work? Also, the tests themselves use a dynamic import and they didn't break previously.